library/MiscUtils.test.cpp

```diff
@@ -0,0 +1,19 @@
+
+#include "MiscUtils.h"
+#include <gtest/gtest.h>
+#include <string>
+
+TEST(MiscUtils, wordwrap) {
+    std::vector<std::string> result;
+
+    word_wrap(&result, "123", 3);
+    ASSERT_EQ(result.size(), 1);
+
+    result.clear();
+    word_wrap(&result, "12345", 3);
```

Comment:
```suggestion
    word_wrap(&result, "123456", 3);
```
probably smarter to have put this on the boundary